src/Game.ts

```diff
@@ -80,6 +83,10 @@ class Game {
             }
         }
     }
+
+    changeTurn(): void {
+        this.turn === 'white' ? (this.turn = 'black') : (this.turn = 'white');
```

Comment:
Możemy to zapisać krócej:
this.turn = this.turn === 'white' ? 'black' : 'white'

Reply:
No tak, już poprawiam :)